### ISSUE TITLE
linter: `unicorn/prefer-math-trunc` incorrectly reports unsigned right shift

### ISSUE THREAD

**Issue by hesprs** [oxc-project/oxc#25239]
### What version of Oxlint are you using?

1.76.0

### What command did you run?

oxlint -A all -D unicorn/prefer-math-trunc repro.ts

### What does your `.oxlintrc.json` (or `oxlint.config.ts`) config file look like?

No config, minimally reproduced in one command.

### What happened?

`repro.ts`:

```TypeScript
declare let value: number;
void (value >>> 0);
void (value >>>= 0);
```

Oxlint reports `error unicorn(prefer-math-trunc): Prefer Math.trunc() over instead of ...` for both `>>>` and `>>>=`. But the canonical [documentation page](https://oxc.rs/docs/guide/usage/linter/rules/unicorn/prefer-math-trunc.html) says it should only report `value | 0`, `~~value`, `value >> 0`, `value << 0` and `value ^ 0`.

It is also logically incorrect, since `>>>` is unsigned, `Math.trunc()` is signed:

```TypeScript
-1 >>> 0          // 4294967295
Math.trunc(-1)    // -1
```
